utilities.tex

```diff
@@ -5,36 +5,27 @@
 \setcounter{section}{1}
 \rSec1[utility]{Utility components}
 
-\ednote{``\tcode{swap}'' is a customization point found by Argument-Dependent Lookup. Due to the nature of
-ADL, the \tcode{swap} in namespace \tcode{std} would be considered if either argument has namespace \tcode{std}
-as an associated type. If \tcode{std::swap} remains unconstrained, then the \tcode{Swappable} concept will
-erroneously return true for some types that are not in fact swappable. Below, we suggest adding constraints
-directly to the overloads of \tcode{swap} in namespace \tcode{std} rather than defining new constrained
-overloads of \tcode{swap} in namespace \tcode{std::experimental::ranges\oldtxt{_v1}}. The precise mechanism by which
-\tcode{swap} is constrained such that it is usable in build environments lacking the concepts language feature
-is unspecified.}
-
+{\color{oldclr}
 \synopsis{Header \tcode{<utility>} synopsis}
 
 \pnum
 Change the \tcode{<utility>} synopsis as follows:
 
 \begin{codeblock}
-@\added{namespace std \{}@
   // ... as before ..
 
   // \ref{utility.swap}, swap:
   template<class T>
-    @\added{requires experimental::ranges\oldtxt{_v1}::Movable<T>()}@
+    @\oldtxt{requires experimental::ranges\oldtxt{_v1}::Movable<T>()}@
```

Comment:
If you're striking this whole section, then just delete these lines instead of marking them as deleted.